crates/web-sys/webidls/enabled/AbstractRange.webidl

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * The origin of this IDL file is
+ * https://dom.spec.whatwg.org/#abstractrange
+ *
+ * Copyright © 2012 W3C® (MIT, ERCIM, Keio), All Rights Reserved. W3C
+ * liability, trademark and document use rules apply.
+ */ 
+
+
+interface AbstractRange {
+  [Throws]
+  readonly attribute Node startContainer;
+  [Throws]
+  readonly attribute unsigned long startOffset;
+  [Throws]
+  readonly attribute Node endContainer;
+  [Throws]
+  readonly attribute unsigned long endOffset;
```

Comment:
I can't find anything in the specification saying that this could throw.
I assume you simply copied this from the Firefox webidl directory, but we tend to follow the specification as close as possible.

Let me know if you have different information.